File: cs50 homework/pset1/pset6/mario.py
import cs50

def get_positive_int():
    print("Height: ", end="")
    while True:
        n = cs50.get_int()
        if n > 0 and n < 24:
            break
        print("Retry: ", end="")
    return n

def pyramid(height):
    for i in range(1, height+1):
        print(" "*(height-i) + "#"*i + "  " + "#"*i)
    return

def main():
    pyramid(get_positive_int())

if __name__ == "__main__":
    main()

File: cs50 homework/pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    printf("Height: ");
    
    int height = get_int();
    
    while (height < 0 || height > 23)
    {
        printf("Height: ");
        height = get_int();
    }
    
    for (int i = 1; i < height+1; i++)
    {
        for (int j = 0; j < height - i; j++)
        {
            printf(" ");
        }
        
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
     
}